### ISSUE TITLE
Completions don't (always) delete their prefix

### ISSUE THREAD

**Issue by Jasper-M** [laughedelic/atom-ide-scala#79]
ide-scala version: 0.9.0
metals version: 0.5.2
scala versions: 2.11.12, 2.12.8
operating systems: OSX, Windows

**Steps to reproduce**
1. open a metals project
2. create file `src/main/scala/Foo.scala`
3. type
```
class Foo {
  java.time.Loc
}
```
4. a popup has automatically appeared by now suggesting `LocalDate`, `LocalTime`, `LocalDateTime`, `Clock`.
5. select `LocalDateTime` with arrows keys and accept
6. text changes to
```
class Foo {
  java.time.LocLocalDateTime
}
```

**Comment by Jasper-M** [comment]
Same thing happens when you accept the first suggested completion with `tab` or `enter`, which is probably the most common situation.

**Comment by cquiroz** [comment]
I've seen this on the previous version as well. I wonder if it is something with the plugin or Atom itself

**Comment by Jasper-M** [comment]
I guess it's some interplay between atom and the plugin. It doesn't happen with completions (based on words previously used) that Atom itself suggests.

**Comment by laughedelic** [comment]
Yes, it's a known problem. I think that this is a bug in atom-languageclient and I'm hoping that this PR will fix it: https://github.com/atom/atom-languageclient/pull/254
